README.md

```diff
@@ -103,25 +104,26 @@ It should connect to Discord and display the currently logged-in application nam
 
 The expected flow for running a bust is:
 
-* Users submit songs into a channel.
-* All users join a voice channel or stage.
-* An admin runs `/list` to list all submitted songs and the order they will be played in.
-* An admin runs `/bust` to start the show. The bot will join the channel and begin playing songs in the order they were submitted.
-* Users comment on songs while they play.
-* An admin can run `/skip` at any time to skip the current song, or `/stop` to manually stop the show.
-* Once the last song has played, the bot will post a concluding message and leave the call.
+- Users submit songs into a channel.
+- All users join a voice channel or stage.
+- An admin runs `/list` to list all submitted songs and the order they will be played in.
+- An admin runs `/bust` to start the show. The bot will join the channel and begin playing songs in the order they were submitted.
+- Users comment on songs while they play.
+- An admin can run `/skip` at any time to skip the current song, or `/stop` to manually stop the show.
+- Once the last song has played, the bot will post a concluding message and leave the call.
 
 ### Command Reference
 
-1. `/list [<channel>]` - Download and list all media sent in the current text channel. Specifying a channel will cause songs to be pulled from that channel instead. This must be run before `/bust`. 
+1. `/list [<channel>]` - Download and list all media sent in the current text channel. Specifying a channel will cause songs to be pulled from that channel instead. This must be run before `/bust`.
 1. `/bust [<song #>]` - Join the vc/stage that the user who ran this command is currently in, and plays the tracks in the channel in order. The user must be in a vc or stage for this to work. Specifying a song index will skip to that index before playing.
 1. `/info` - Show some statistics about currently listed songs.
+1. `/preview [<attachment>] [<submit_message>]` - Show a preview of a file's artist, title and artwork.
```

Comment:
There should probably be some indication that [<submit_message>] is optional like in the description for the /list command, though maybe we should eventually add a different notation for this

Reply:
In the notation we currently have (and it standard across cli tools to the best of my knowledge) `[]` indicate an optional argument.

So in fact in this case the surrounding `[]`'s should be removed from `<attachment>`, since it is a required argument.

Reply:
In our current notation we seem to just use `[<arg>]` for all arguments, though I agree that the `[]` should be removed for non-optional ones. See the `\image url <url>` command etc where `url` is not optional. I also just noticed that the `\image view` and `\image clear` commands are listed as having arguments, which is just false lol